state_container_py/examples/reducer.py

```diff
@@ -1,10 +1,11 @@
+from state_container_py.base_state import BaseState
 from state_container_py.reducer import Reducer
 
 
 # TODO: Deal with defaults
-class TodoState:
-    name: str
-    thing_to_do: str
+class TodoState(BaseState):
```

Comment:
Need to think about how to ask users to submit their user defined state to the store.